host/3.0/buster/amd64/java/java11/java11.Dockerfile

```diff
@@ -19,6 +20,10 @@ RUN EXTENSION_BUNDLE_VERSION=1.8.1 && \
     EXTENSION_BUNDLE_FILENAME=Microsoft.Azure.Functions.ExtensionBundle.1.8.1_linux-x64.zip && \
     apt-get update && \
     apt-get install -y gnupg wget unzip && \
+    wget https://aka.ms/download-jdk/microsoft-jdk-${JAVA_VERSION}-linux-x64.tar.gz && \
+    mkdir -p /usr/lib/jvm && \
+    tar xvzf microsoft-jdk-${JAVA_VERSION}-linux-x64.tar.gz -C /usr/lib/jvm &&\
+    rm -f microsoft-jdk-${JAVA_VERSION}-linux-x64.tar.gz && \
```

Comment:
Please investigate the images you generate to ensure that only necessary information is copied over. I think you should be fine. But be on the lookout for anything in this folder that is related to another runtime or architecture.

Reply:
Random unneeded bits have been sitting in a lot of our docker images.  For example, we used to have ext bundles for linux, mac, and windows in many diff architectures. Not a huge deal just balloons image size

Reply:
Yes, so the binaries will have one folder once unzip it and this entire folder will be set as our java_home.